doc/march_packages/march_hardware_interface.rst

```diff
@@ -26,23 +26,50 @@ no allocations take place during real time.
 .. _ros_control: https://wiki.ros.org/ros_control
 .. _realtime_tools: https://wiki.ros.org/realtime_tools
 
+Controllers
+^^^^^^^^^^^
+Currently, the hardware interface can use two types of controllers: position
+and effort controllers. Effort control is currently the default. The
+march_hardware_interface package is also responsible for launching the
+controllers and providing their configurations. These configurations are stored
+in the :hardware-interface:`config <march_hardware_interface/config>` directory.
+These contain settings for the PID controllers and the joint trajectory action
+server. See the `Controllers`_ section on the ros_control wiki for more info on
+what type of controllers exists and how to configure them.
+
+.. _controllers: https://wiki.ros.org/ros_control#Controllers
+
+Safety Limits
+^^^^^^^^^^^^^
+Besides controlling the motors, the hardware interface also enforces safety
+limits. For this it uses the ``joint_limits_interface`` from ros_control. The
+robot has so called *soft limits*, which provide a buffer for the actual
+mechanical limits of the joints. The soft limit values are loaded from the URDF.
+The job of the limits interface is to limit the output of the controllers as to
+not exceed the joint limits defined in the URDF. Enforcing the limits occurs at
```

Comment:
I think it would be useful to link https://wiki.ros.org/pr2_controller_manager/safety_limits around here; it contains an explanation of how the safety limits work.